packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -8,11 +8,10 @@
   "scripts": {
     "build": "backstage-cli plugin:build",
     "lint": "backstage-cli lint",
-    "test": "backstage-cli test", 
+    "test": "backstage-cli test",
     "clean": "backstage-cli clean"
   },
   "devDependencies": {
-    "@backstage/cli": "^{{version}}",
```

Comment:
Sorry, meant duplicates between `devDependencies` / `dependencies`. Every package should still list its own dependencies even if they're already a part of the root, for portability and making sure things are verified on updates. And maybe some other reasons that I forgot :grin:.

Although I guess it's not as important for dev dependencies, since the package won't be built outside of this repo. It would be a bit convenient to only have to update the cli in a single place, but I'm concerned it could cause issues with versioning and things not being properly retested in lerna when the root version is bumped.

What do ppl think? I'd lean towards caution and keeping it in every package.

Reply:
I agree with you on being more cautious.

I have only used lerna briefly in a couple of small projects and dev dependencies at the root were convenient in that scenario but those weren't open source projects so easy to get away with there :)

I'll change it back to having all dev dependencies in the package for this PR.